config/locales/affordability.en.yml

```diff
@@ -42,15 +42,17 @@ en:
         mortgage_calculator/person:
           raw_annual_income: "Your annual income"
           raw_monthly_net_income: "Your monthly take-home pay"
-          annual_income: "Your annual income"
-          extra_income: "Any other annual income"
-          monthly_net_income: "Your monthly take-home pay"
+          annual_income: "What is your annual income or salary before tax?"
+          extra_income: "Do you have any other income? (Optional)"
+          monthly_net_income: "What is your monthly take-home pay?"
+          monthly_net_sub: "(this is your monthly income after tax, minus pension, NI and other deductions)"
```

Comment:
This copy is wrong when used for the second applicant: should be "this is their ..." not "this is your ..."

Reply:
Fixed